src/main/java/com/example/board/converter/PostConverter.java

```diff
@@ -0,0 +1,27 @@
+package com.example.board.converter;
+
+import com.example.board.domain.Post;
+import com.example.board.dto.PostDto;
+
+import java.time.LocalDateTime;
+
+public class PostConverter {
+    public static Post convertFromDtoToPost(PostDto postDto) {
```

Comment:
인자로도 무엇으로부터 변환하는지 유추가능해서 다음처럼 메소드명을 단축시켜볼 수 있을거 같아요 🤔🤔

```suggestion
    public static Post convertToPost(PostDto postDto) {
```

Reply:
확실히 이정도로도 충분히 의미가 파악이 가능한 것 같습니다.
제안해 주신 대로 수정하였습니다!